source/httpfs/partial_driver.go

```diff
@@ -136,3 +137,20 @@ func (p *PartialDriver) ReadDown(version uint) (r io.ReadCloser, identifier stri
 		Err:  os.ErrNotExist,
 	}
 }
+
+func (p *PartialDriver) open(path string) (http.File, error) {
+	f, err := p.fs.Open(path)
+	if err == nil {
+		return f, nil
+	}
+	// Some non-standard file systems may return errors that don't include the path, that
+	// makes debugging harder.
+	if !errors.As(err, new(*os.PathError)) {
```

Comment:
Use `errors.Is()` if the target is unused.

Reply:
`errors.Is()` has different semantic - exact value match. I need just type match.